### DISCUSSION TITLE
Using third party packages with Zero

### DISCUSSION THREAD

**Discussion by binaryfire** [laravel-zero/laravel-zero#463] [Q&A] [answered]
Hi all,

Are there any caveats when using third party packages with Laravel Zero? For example `saloonphp/saloon`, `symfony/uid` etc.

Mainly wondering about Box and the build process.

**Comment by owenvoke** [comment, marked as answer]
Most packages should be fine. Just need to register the service provider as they aren't autoloaded. 👍🏻 I haven't tried Saloon, but I don't see why it wouldn't work. 🤷🏻

Symfony UID should work fine too. As long as the `vendor` files are included, they should work fine.
